PurchasesTests/Networking/BackendTests.swift

```diff
@@ -1502,7 +1502,7 @@ class BackendTests: XCTestCase {
 
         let currentAppUserID = "old id"
         let underlyingErrorMessage = "header fields too large"
-        let underlyingErrorCode = "123456"
+        let underlyingErrorCode = BackendErrorCode.cannotTransferPurchase.rawValue
```

Comment:
"123456" would be converted to `.unknownError` with the change from line https://github.com/RevenueCat/purchases-ios/pull/670/files#diff-c4fc22c0d43f6801ea744bd065af825839de0cbe05fdc979640850bcee2135afR278